sdcm/cluster.py

```diff
@@ -727,6 +729,32 @@ def destroy(self):
         self.stop_nemesis()
         super(ScyllaCluster, self).destroy()
 
+    def update_db_binary(self, node_list=None):
+        if node_list is None:
+            node_list = self.nodes
+
+        new_scylla_bin = self.params.get('update_db_binary')
+        if new_scylla_bin != '':
```

Comment:
Question: What does self.params.get('update_db_binary') return if user does not specify update_db_binary in the config file, None?